index.js

```diff
@@ -628,6 +628,50 @@ function modResponse(response, extended) {
   return response
 }
 
+function countResponseFields(resp) {
+  const counts = {};
+
+  if (resp.data) {
+    countResponseFieldsRecursive(counts, "data", resp.data);
+  }
+
+  counts.errors = resp.errors ? resp.errors.length : 0;
```

Comment:
is this line needed? In the follow-up PR, errors are passed into the sidecar as they are anyway